pandas/tests/arrays/test_datetimelike.py

```diff
@@ -1,13 +1,50 @@
 # -*- coding: utf-8 -*-
 import numpy as np
+import pytest
 
 import pandas as pd
+import pandas.util.testing as tm
 
 from pandas.core.arrays.datetimes import DatetimeArrayMixin
 from pandas.core.arrays.timedeltas import TimedeltaArrayMixin
 from pandas.core.arrays.period import PeriodArrayMixin
 
 
+# TODO: more freq variants
+@pytest.fixture(params=['D', 'B', 'W', 'M', 'Q', 'Y'])
+def period_index(request):
```

Comment:
shouldn't this rather be called 'period_array' ? (since this is in the array tests directory, and is meant to later become the actual array tests?)

Reply:
> shouldn't this rather be called 'period_array' 

No, the fixture is specifically returning a PeriodIndex.  I imagine at some point when these are more fleshed out they'll go up in conftest, or parts might go in pd.util.testing